src/resources/resources.controller.ts

```diff
@@ -0,0 +1,74 @@
+import {
+    Controller,
+    Get,
+    Post,
+    Body,
+    Patch,
+    Param,
+    Delete,
+    ParseIntPipe,
+} from "@nestjs/common";
+import { ApiOperation, ApiTags } from "@nestjs/swagger";
+import { ResourcesService } from "./resources.service";
+import { CreateResourceDto } from "./dto/create-resource.dto";
+import { UpdateResourceDto } from "./dto/update-resource.dto";
+import { DeleteResourceDto } from "./dto/delete-resource.dto";
+
+@Controller("team-resources")
+@ApiTags("team resources")
+export class ResourcesController {
+    constructor(private readonly resourcesService: ResourcesService) {}
+
+    @ApiOperation({
+        description:
+            "Adds a URL with title to the team's resources, addedBy: teamMemberId (int)",
+    })
+    @Post(":teamMemberId")
```

Comment:
How likely is it for frontend to store teamMemberId? will it be stored in localstorage etc? Another concern is when people participate in multiple teams, they would have multiple `teamMemberId` so (teamId + userId) would be the best way to gaurantee uniqueness from the frontend (User Id would be in the auth header, TeamId would probably already be in the url params, both of these are readily available). We have been using userId + teamId in other endpoints (since teamId would be somewhere in the frontend because a lot of other things use it)

Reply:
Should I change this to @Post(":teamId/:userId") then? Couldn't find an example like this searching the codebase.

Reply:
would be teams/:teamId. userId would come from the auth header, but for now it can be in the body
both ideation and tech stack endpoints have something similar to this